src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -20,41 +20,33 @@ use super::{LazyBatchColumn, VectorValue};
 use crate::coprocessor::codec::mysql::Tz;
 use crate::coprocessor::codec::Result;
 
-pub struct BatchRows<E> {
-    /// Multiple interested columns. Each column is either decoded, or not decoded.
+/// Stores multiple `LazyBatchColumn`s. Each column have equal length.
```

Comment:
```suggestion
/// Stores multiple `LazyBatchColumn`s. Each column has an equal length.
```

Reply:
Done, addressed in commit https://github.com/tikv/tikv/pull/4215/commits/f235470d313e8f5e3b4b3e422eaecaf7883ca3df of https://github.com/tikv/tikv/pull/4215